src/app/menu/menu.component.html

```diff
@@ -14,26 +14,36 @@
       <ul class="navbar-nav ml-auto">
 
         <li class="nav-item">
-          <a href="" class="nav-link">Home</a> <!-- Welcome to football app -->
+          <a href="/home" class="nav-link">Home</a> <!-- Welcome to football app -->
         </li>
         <li class="nav-item dropdown">
-          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
+          <a class="nav-link dropdown-toggle" href="#" id="navDrop1" role="button"
              data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
             Top&nbsp;5&nbsp;Leagues
```

Comment:
Myślę, że to samo co poniżej no chyba że top 5 leagues to stała decyzja a nie tymczasowa? Dodatkowo po co wszędzie spacje nierozrywające &nbsp; ?